src/common/libjob/jj.c

```diff
@@ -16,6 +16,8 @@
 #include <string.h>
 #include <jansson.h>
 
+#include "src/common/libccan/ccan/str/str.h"
```

Comment:
`-I$(top_srcdir)/src/common/libccan` is specified in the Makefile, so only `ccan/str/str.h` is required here.